app/views/chapters/show.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="row mt-5 mb-5">
+  <div class="col-md-4">
+    <p><%= t "name_chapter" %>: <%= @chapter.name %></p>
+    <%= image_tag @chapter.image.url if @chapter.image? %>
+    <p><%= t "content" %></p>
+    <p><%= @chapter.content %></p>
+  </div>
+</div>
```

Comment:
bỏ file này đi nhé, vì bây giờ chưa dùng đến. 
cái view show này để hiển thị nội dung chương toàn màn hình cho user đọc ông ạ.  
bớt đc 1 file change.